app/app.js

```diff
@@ -96,11 +105,25 @@ app.use(helmet.ieNoOpen())
 app.use(helmet.hidePoweredBy({setTo: 'Whisky Powered.'}))
 app.use(helmet.contentSecurityPolicy({
     directives: {
-        defaultSrc: ["'self'", "https://stackpath.bootstrapcdn.com"],  // default value for all directives that are absent
-        scriptSrc: ["'self'", "https://code.jquery.com/", "https://stackpath.bootstrapcdn.com", "https://cdnjs.cloudflare.com"],   // helps prevent XSS attacks
+        defaultSrc: [
+"'self'",
+"https://stackpath.bootstrapcdn.com"
+],  // default value for all directives that are absent
+        scriptSrc: [
+"'self'",
```

Comment:
What is this `self` about

Reply:
Refers to "self" as in if the, in this case, script, comes from your own server or domain it's accepted under CSP, this was part of security PR

Reply:
Could u add a comment?